app/src/main/res/layout/offer_header.xml

```diff
@@ -73,18 +73,23 @@
 
             <androidx.constraintlayout.widget.ConstraintLayout
                 android:id="@+id/startDateContainer"
+                style="?materialButtonTextStyle"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_margin="@dimen/base_margin_double"
-                android:background="@drawable/background_pick_date"
+                android:background="@drawable/background_rounded_corners_ripple"
                 android:padding="@dimen/base_margin">
 
                 <TextView
                     android:id="@+id/startDateLabel"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:text="@string/START_DATE_BTN"
+                    android:drawablePadding="@dimen/base_margin"
+                    android:gravity="center_vertical"
+                    android:text="@string/OFFER_ACCESS_DATE"
```

Comment:
This text will change depending on if the user is in moving flow or "offer" flow. Logic for that is not the scope for this PR, and is for now hard coded to use moving flow text only.